torch/csrc/jit/codegen/cuda/lower_index.cpp

```diff
@@ -221,6 +221,83 @@ void IndexLowering::handle(const IndexSelectOp* sop) {
   GpuLower::current()->propagateExprInfo(sop, back());
 }
 
+namespace {
+Val* packToRealIndices(kir::TensorIndex* idx) {
+  Val* real_indices = nullptr;
+  bool first = true;
+  for(auto x : idx->indices()) {
+    if(first) {
+      real_indices = x;
+      first = false;
+    }
+    else {
+      real_indices = SimplifyingIrBuilder::addExpr(real_indices, x);
```

Comment:
Nit. A compact way to do this may be 
`real_indices = SimplifyingIrBuilder::addExpr( real_indices, x)` since the underneath implementation of `addExpr` will handle the corresponding condition logic
<img width="908" alt="image" src="https://user-images.githubusercontent.com/407784/205321061-17b9ed34-3839-4327-be9d-4401b79b8c6e.png">

Reply:
Thanks. Got it.